First/dailyExercises/LargestNum.java

```diff
@@ -0,0 +1,21 @@
+package firstPackage.First.dailyExercises;
+
+public class LargestNum {
+    public static void main(String[] args) {
+        int numFirst = 10;
+        int numSecond = 250;
```

Comment:
this is not medium complexity code, please update as per that

Reply:
Done. Updated the code